data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/GetStateFinalityCheckpointsTest.java

```diff
@@ -28,42 +26,31 @@
 import tech.pegasys.teku.datastructures.state.BeaconState;
 import tech.pegasys.teku.datastructures.util.DataStructureUtil;
 import tech.pegasys.teku.infrastructure.async.SafeFuture;
-import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 
 public class GetStateFinalityCheckpointsTest extends AbstractBeaconHandlerTest {
   final DataStructureUtil dataStructureUtil = new DataStructureUtil();
   final BeaconState state = dataStructureUtil.randomBeaconState();
 
-  @Test
-  public void shouldReturnUnavailableWhenStoreNotAvailable() throws Exception {
-    final GetStateFinalityCheckpoints handler =
-        new GetStateFinalityCheckpoints(chainDataProvider, jsonProvider);
-    when(context.pathParamMap()).thenReturn(Map.of("state_id", "head"));
-    when(chainDataProvider.isStoreAvailable()).thenReturn(false);
-
-    handler.handle(context);
-    verifyStatusCode(SC_NOT_FOUND);
-  }
```

Comment:
Assuming we don't need this because testing for pre-genesis is now just covered by the state selector which would return empty.

Reply:
That's my belief 👍